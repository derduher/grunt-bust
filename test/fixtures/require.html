<!DOCTYPE html>
<html>
<head>
<script>
require(['test/fixtures/common.js'], function () {
    require(['index']);
});
</script>
</head>
</html>
