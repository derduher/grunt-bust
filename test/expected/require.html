<!DOCTYPE html>
<html>
<head>
<script>
require(['test/fixtures/common.cbuster-6030bec2badaea52a7c1e25f7685a88d.js'], function () {
    require(['index.cbuster-68176f2823a2e07bf591afd341a8d8ab']);
});
</script>
</head>
</html>
